fiftyone/core/collections.py

```diff
@@ -10601,7 +10750,15 @@ def _parse_aggregations(self, aggregations, allow_big=True):
         big_aggs = {}
         batch_aggs = {}
         facet_aggs = {}
+        stream = None
+
         for idx, aggregation in enumerate(aggregations):
+            # stream is True if all aggregations are lazy
+            if lazy := getattr(aggregation, "_lazy", False):
```

Comment:
More code smell. `_lazy` is not defined on the `Aggregation` interface.

This usage pattern would ideally be documented on the `Aggregation` class via attributes, like `_has_big_result`, `_is_big_batchable`, etc.